hack/install-konveyor.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Figure out where we are being run from.
+# This relies on script being run from:
+#  - ${PROJECT_ROOT}/hack/install-tackle.sh
+#  - ${PROJECT_ROOT}/bin/install-tackle.sh
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__root="$(cd "$(dirname "${__dir}")" && pwd)"
+__repo="$(basename "${__root}")"
+__bin_dir="${__root}/bin"
+__os="$(uname -s | tr '[:upper:]' '[:lower:]')"
+__arch="$(uname -m | sed 's/x86_64/amd64/' | sed 's/aarch64/arm64/')"
+
+# Update PATH for execution of this script
+export PATH="${__bin_dir}:${PATH}"
+
+NAMESPACE="${NAMESPACE:-konveyor-tackle}"
+OPERATOR_BUNDLE_IMAGE="${OPERATOR_BUNDLE_IMAGE:-quay.io/konveyor/tackle2-operator-bundle:latest}"
+TACKLE_CR="${TACKLE_CR:-}"
+
+if ! command -v kubectl >/dev/null 2>&1; then
+  kubectl_bin="${__bin_dir}/kubectl"
+  mkdir -p "${__bin_dir}"
+  curl -Lo "${kubectl_bin}" "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/${__os}/${__arch}/kubectl"
+  chmod +x "${kubectl_bin}"
+fi
+
+if ! command -v operator-sdk >/dev/null 2>&1; then
+  operator_sdk_bin="${__bin_dir}/operator-sdk"
+  mkdir -p "${__bin_dir}"
+
+  version=$(curl --silent "https://api.github.com/repos/operator-framework/operator-sdk/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+  curl -Lo "${operator_sdk_bin}" "https://github.com/operator-framework/operator-sdk/releases/download/${version}/operator-sdk_${__os}_${__arch}"
+  chmod +x "${operator_sdk_bin}"
+fi
+
+run_bundle() {
+  kubectl auth can-i create namespace --all-namespaces
+  kubectl create namespace ${NAMESPACE} || true
+  operator-sdk run bundle ${OPERATOR_BUNDLE_IMAGE} --namespace ${NAMESPACE}
+
+  # If on MacOS, need to install `brew install coreutils` to get `timeout`
+  timeout 600s bash -c 'until kubectl get customresourcedefinitions.apiextensions.k8s.io tackles.tackle.konveyor.io; do sleep 30; done' \
+  || kubectl get subscription --namespace ${NAMESPACE} -o yaml konveyor-operator # Print subscription details when timed out
+}
+
+install_tackle() {
+  echo "Waiting for the Tackle CRD to become available"
+  kubectl wait \
+    --namespace ${NAMESPACE} \
+    --for=condition=established \
+    customresourcedefinitions.apiextensions.k8s.io/tackles.tackle.konveyor.io
+
```

Comment:
So looking at testdata folder of operator-sdk I think this remains an anti-pattern.

https://github.com/operator-framework/operator-sdk/tree/master/testdata/helm/memcached-operator/config/crd

`config/` folder and kustomize still used. Is the intent to break away from operator-sdk helm pattern?

Reply:
This PR essentially mean  going back to manually annotating CSV in bundle/ and the main pain point for me is converting into JSON the alm-examples which `config/samples` solves.

https://sdk.operatorframework.io/docs/building-operators/helm/quickstart/

Reply:
CRDs are now in `./helm/templates` which is not the case in `https://github.com/operator-framework/operator-sdk/tree/master/testdata/helm/memcached-operator/helm-charts/memcached/templates`.

Just noting the anti-pattern. If everyone is fine with that, then its a non-issue.

Staying in pattern could make future migrations to https://github.com/operator-framework/rukpak (supersede operator-sdk) easier.

Reply:
My understanding is that memcached-operator is a helm based operator...meaning the operand is being deployed using some variation of `helm install`. Here we are just trying to make it easier to generate the operator manifests because the customization of `kustomize` manifests is so cumbersome (specifically the images that are inside environment variables `RELATED_IMAGES_`). The other really painful thing is computing and replacing skips and replaces.

Basically, we want to be able to helm install the operator (that is ansible based). I could rename `./helm` to `./config` but I think that may confuse people looking at the project.

Lastly, the only real intelligence I see from operator-sdk is in taking a bunch of YAMLs on stdin and generating the `./bundle` dir. We aren't leveraging any of the controller-gen tooling that would make this more difficult.

Reply:
> because the customization of kustomize manifests is so cumbersome (specifically the images that are inside environment variables RELATED_IMAGES_

got it.

> memcached-operator is a helm based operator.

They have [helm](https://github.com/operator-framework/operator-sdk/tree/master/testdata/helm/memcached-operator), [go/v3](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3), and [go/v4-alpha](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v4-alpha) variants of memcached operator.

> computing and replacing skips and replaces.

If you own the catalogsource, [opm tooling makes it pretty easy](https://github.com/kaovilai/oadp-operator/blob/edge/appendMakefile#L2) to add new bundle as a replacement of current one.